src/hooks/api/authenticated-query.ts

```diff
@@ -183,12 +183,21 @@ const isDuplicateMutation = (
     return false;
   }
 
+  if (mutationKey === 'request-ownership') {
+    return false;
+  }
```

Comment:
Since we're starting to have a lot of these I would perhaps make it an array then we can do `disabledMutations.includes(mutationKey)` or something